lib/bitex_bot/models/api_wrappers/bitstamp/bitstamp_api_wrapper.rb

```diff
@@ -31,8 +31,8 @@ def self.cancel(order)
     raise ApiWrapperError, "Bitstamp cancel! failed: #{e.message}"
   end
 
-  def self.find_lost(type, price)
-    orders.find { |o| o.order_method == type && o.price == price && o.datetime.to_datetime >= 5.minutes.ago.to_datetime }
+  def self.find_lost(type, price, _quantity)
+    orders.find { |o| o.type == type && o.price == price && o.timestamp >= 5.minutes.ago.utc.to_i }
```

Comment:
`.utc` is not necessary here, `.to_i` returns a [UNIX Time](https://en.wikipedia.org/wiki/Unix_time)